rclcpp/include/rclcpp/parameter.hpp

```diff
@@ -86,6 +86,12 @@ class ParameterVariant
     value_.string_value = string_value;
     value_.type = rcl_interfaces::msg::ParameterType::PARAMETER_STRING;
   }
+  explicit ParameterVariant(const std::string & name, const char * string_value)
+  : name_(name)
+  {
+    value_.string_value = std::string(string_value);
+    value_.type = rcl_interfaces::msg::ParameterType::PARAMETER_STRING;
+  }
```

Comment:
Please call the constructor with the std::string argument instead of reimplementing the same body.